src/components/order-service/order-service.controller.ts

```diff
@@ -164,6 +164,12 @@ export class OrderServiceController implements angular.IController {
     this.ctrl.nextTitle = 'Create';
     this.reviewStep.allowed = this.bindStep.valid;
 
+    if (this.isNewProject()) {
+      this.ctrl.projectDisplayName = this.ctrl.selectedProject.metadata.annotations['new-display-name'] || this.ctrl.selectedProject.metadata.name;
+    } else {
+      this.ctrl.projectDisplayName = this.$filter('displayName')(this.ctrl.selectedProject);
+    }
+
```

Comment:
So this PR seems to be a little more than just a version bump.  What are these other changes? -thanks